taxonomies/nutrients.txt

```diff
@@ -5793,7 +5799,7 @@ pi:क्याल्सियम
 pl:Wapń
 # pms:Càuss
 # pnb:کیلشیم
-pt:Cálcio
+pt:Cálcio, Ca2+, Ca
```

Comment:
Same as before, is there any product specifying "Ca2+" or "Ca" as a nutrient?

Reply:
Please see the previous image.